.github/workflows/localize.yml

```diff
@@ -0,0 +1,34 @@
+name: Localize
+
+on: 
+  push:
+    branches:
+      - master
+  # Also run every day at midnight to pick up new localizations
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 0 * * *'
+
+jobs:
+  Localize:
+    runs-on: macos-10.15
```

Comment:
is there a way we can have this specified in only one place (I think other scripts use this?)

Reply:
I've spent some time looking through this and don't think we can abstract the runs-on variable. We could look into moving the localization step into the ci.yml file, and have it conditionally run for pushes to master with an [if statement](https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions), but for now I want to keep them separate for our own sanity standing this up and not wanting to break our CI.

As of right now, it's duplicated between this file and `.gitub/workflows/ci.yml`. While the same string shows up in the `.ado` folder, the fact that ADO Pipelines and GitHub Actions use the same VM image name isn't guaranteed to be true going forward.